### ISSUE TITLE
procmon plugin fails on Python 3.5 when handling process output

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8919]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)| @moshez reported|
|-|-|
|Trac ID|trac#8919|
|Type|defect|
|Created|2016-11-25 18:00:55Z|

Whenever the process run writes to stdout, an exception is produced.

```
twist --log-file=build/procmon.issue procmon ./foo 
```

Attachments:

* [foo](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/794/794c24b654b9ab3a06df0557e5f643ee5308dc8f/0beec7b5ea3f0fdbc95d0dd47f3c5bc275da8a33) (30 bytes) - added by moshez on 2016-11-25 18:01:57Z - Script to cause problem
* [procmon.issue](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/794/794c24b654b9ab3a06df0557e5f643ee5308dc8f/22a083aea85eab05728a8c953978d48d2b8b736e.issue) (24525 bytes) - added by moshez on 2016-11-25 18:02:46Z - Log with problem

<details><summary>Searchable metadata</summary>

```
trac-id__8919 8919
type__defect defect
reporter__moshez moshez
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__runner runner
keywords__None None
time__1480096855395233 1480096855395233
changetime__1480559626474368 1480559626474368
version__None None
owner__moshez moshez

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)<a name="note_1"></a>|@moshez set owner to @moshez<br>@moshez set status to `assigned`|
|-|-|

https://github.com/twisted/twisted/pull/607

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)|@moshez removed owner<br>@moshez set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_3"></a>|@wsanchez commented|
|-|-|

reviewed, good to go.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez set owner to @wsanchez<br>@wsanchez set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez set owner to @moshez<br>@wsanchez set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_7"></a>|@glyph set status to `closed`|
|-|-|

In changeset 5cda954e9613d7ecd9a1f21e8f760a654571a7aa
```
#!CommitTicketReference repository="" revision="5cda954e9613d7ecd9a1f21e8f760a654571a7aa"
Merge pull request #607 from moshez/8919-procmon-py35-fix

Author: moshez

Reviewer: wsanchez

Fixes: #8919

On Python 3, procmon now handles process output without exceptions
```
